pkg/cmd/pipeline/start.go

```diff
@@ -81,6 +82,12 @@ type startOptions struct {
 	UseParamDefaults   bool
 	TektonOptions      flags.TektonOptions
 	PodTemplate        string
+	LocalToWorkspace   []LocalToWorkspace
+}
+
+type LocalToWorkspace struct {
+	Name    string
+	DirPath string
```

Comment:
should we take storage size and access mode as well in input as AccessModes may vary depending on which platform cluster is running. Ref:- https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes